Mathlib/RingTheory/UniqueFactorizationDomain/NormalizedFactors.lean

```diff
@@ -162,9 +163,9 @@ theorem normalizedFactors_mul {x y : α} (hx : x ≠ 0) (hy : y ≠ 0) :
 theorem normalizedFactors_pow {x : α} (n : ℕ) :
     normalizedFactors (x ^ n) = n • normalizedFactors x := by
   induction' n with n ih
-  · simp
+  · simp [zero_nsmul]
   by_cases h0 : x = 0
-  · simp [h0, zero_pow n.succ_ne_zero, smul_zero]
+  · simp [h0, zero_pow n.succ_ne_zero, nsmul_zero]
```

Comment:
Nothing needs changing here but do you or anyone else know why some smuls had to be changed to nsmuls in this PR?